tasks/flow_1.ts

```diff
@@ -0,0 +1,15 @@
+import {task} from "hardhat/config";
+
+task("flow_1", "Deploy 2 tokens and factory")
```

Comment:
Can we somehow make names more description, i would not know what flow_1 means

Reply:
done